### ISSUE TITLE
Add to F-Droid

### ISSUE THREAD

**Issue by OmlineEditor** [dovecoteescapee/ByeDPIAndroid#14]
Please add your application to the catalog on the website https://f-droid.org/

**Comment by IzzySoft** [comment]
Would help if some screenshots are available (or even full fastlane structures, which F-Droid would ask for). Library scan looks promising, no "offending libs". Only thing detected by the scanners at IzzyOnDroid so far is:

```
SigningBlock blobs:
-------------------
0x504b4453 (DEPENDENCY_INFO_BLOCK; GOOGLE)
```

which F-Droid usually does not scan for (IoD does), but could be easily avoided with a tiny addition to the `build.gradle`:

```gradle
android {
    dependenciesInfo {
        // Disables dependency metadata when building APKs.
        includeInApk = false
        // Disables dependency metadata when building Android App Bundles.
        includeInBundle = false
    }
}
```

For some background: that BLOB is supposed to be just a binary representation of your app's dependency tree. But as it's encrypted with a public key belonging to Google, only Google can read it – and nobody else can even verify what it really contains. More details can be found e.g. here: [Ramping up security: additional APK checks are in place with the IzzyOnDroid repo](https://android.izzysoft.de/articles/named/iod-scan-apkchecks#blobs).

With screenshots being available, I could go adding the app to IzzyOnDroid within less than 24 hours (with fastlane structures being established as a side-effect; those I could bring here then via PR). Ideally, that blob should then be addressed with the next release :wink:

Looking forward to your reply :smiley:

**Comment by dovecoteescapee** [comment]
@IzzySoft 

> but could be easily avoided with a tiny addition to the `build.gradle`

Done

> Would help if some screenshots are available

I added a preview
https://github.com/dovecoteescapee/ByeDPIAndroid/blob/master/.github/images/preview.png

**Comment by IzzySoft** [comment]
Thanks! Just started to add it to IoD, but got stopped in the tracks by [VT reporting](https://www.virustotal.com/gui/file/614501a40692bb3182dd27e95501ed4b95e510bdaafddb213b0b65ca6b4e4391/detection/f-614501a40692bb3182dd27e95501ed4b95e510bdaafddb213b0b65ca6b4e4391-1723468968) "16 malware alerts" :scream: Could you please check what's going on there? No offense meant of course, but something strange got dragged in it seems…

**Comment by dovecoteescapee** [comment]
Yes, I know and I'm working on it #61. I'll text you when I fix it

**Comment by IzzySoft** [comment]
Thanks! Looking forward to your ping then.

**Comment by dovecoteescapee** [comment]
@IzzySoft please check the latest version

**Comment by IzzySoft** [comment]
Looks good, thanks! Adding it now. If you had some screenshots that would be good – you know, pictures often explain things better and give a better idea.

**Comment by dovecoteescapee** [comment]
@IzzySoft I added some screenshots but the interface is very boring
https://github.com/dovecoteescapee/ByeDPIAndroid/tree/master/.github/images

**Comment by IzzySoft** [comment]
Thanks! Added. If you want the metadata here in fastlane format (so you can define how your app is presented), let me know – I can send it over via PR, and you're welcome using the [IzzyOnDroid Fastlane Documentation](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Fastlane) to build on it then.

**Comment by dovecoteescapee** [comment]
@IzzySoft Thank you! It will be great if you send PR with Fastlane

**Comment by IzzySoft** [comment]
There you go: #100 :smiley:

**Comment by eternal-sorrow** [comment]
Any progress on this?

**Comment by linsui** [comment]
@dovecoteescapee Do you want to add it to f-droid.org? I can help package it.

**Comment by eternal-sorrow** [comment]
And silence...

**Comment by mckaygerhard** [comment]
@eternal-sorrow  noted? very rare that @OmlineEditor started the issue but later just dissapear.. based on their activity.. seems its a fake user to open issues to give importance.. it opens issues in russian and chinesse langs but once opened does not reply anymore

in any case i also ask for progress on this issue!

**Comment by OmlineEditor** [comment]
> very rare that @OmlineEditor started the issue but later just dissapear.. based on their activity.

That's right, I'm a Chinese bot that my owner forgot to launch me on his server and I went to sleep and was offline for several months.

**Comment by mckaygerhard** [comment]
> > very rare that @OmlineEditor started the issue but later just dissapear.. based on their activity.
> 
> That's right, I'm a Chinese bot that my owner forgot to launch me on his server and I went to sleep and was offline for several months.

the idea is to take serously this.. if other ask for status you can support it like i do .. take this seriously.. cos really looks like a bot.. 

i hope Fdroi got more solutions in their repos.. we need real free software and not more confusion  of open source acting like free software solutions

**Comment by eternal-sorrow** [comment]
@mckaygerhard The opener of the issue has nothing to do with this. If she didn't open this issue, i would do it. I want reply from the app developers, not from the issue opener.
